Libraries/Components/AccessibilityInfo/AccessibilityInfo.ios.js

```diff
@@ -85,6 +85,13 @@ const AccessibilityInfo = {
     });
   },
 
+  /**
+   * macOS only
+   */
+  isHighContrastEnabled: function(): Promise<boolean> {
```

Comment:
should we call it hightcontrast when for apple platform it is increased contrast? high contrast is pretty Window's term right?

Reply:
I'm happy to be shown I'm wrong here but I get the impression from [Apple's support docs](https://support.apple.com/lv-lv/guide/mac-help/unac089/mac) that `Increase Contrast` is the overarching feature encapsulating many different contrasts whereas `HighContrast` is one of many contrast options within it (e.g. `NSAppearanceNameAqua`, `NSAppearanceNameDarkAqua`, `NSAppearanceNameAccessibilityHighContrastDarkAqua`, `NSAppearanceNameVibrantDark
`, `NSAppearanceNameAccessibilityHighContrastVibrantDark`).

If other platforms call it "High Contrast" then I think we just have to choose the one that makes the most sense to go by because I get the impression it won't match up perfectly for all platforms.

Reply:
Since we're writing this feature primarily for macOS, I'd also lean towards IncreaseContrast, but again am happy to see how I'm misunderstanding Apple's docs :)

Reply:
We also plan to implement this API on iOS, Android, and Windows, which already use "high contrast" terminology in their APIs (i.e. `UIAccessibilityContrastHigh` on iOS, `isHighTextContrastEnabled()` on Android, `HighContrast()` on Windows and the `-ms-high-contrast` media query in Edge). I really think "high contrast" is right choice for this API.

Reply:
Yea the main argument against using the macOS terminology is to keep it consistent cross-plat. I can see why that's important enough to keep it